core/src/subscription/quic_stream.rs

```diff
@@ -125,10 +125,10 @@ impl Trackable for TrackedQuic {
 
             if let Some(long_header) = &quic_clone.long_header {
                 if long_header.dcid_len > 0 {
-                    self.connection_id.insert(long_header.dcid.clone());
+                    self.connection_id.insert(&long_header.dcid);
```

Comment:
This results in a mismatched type, expected `String`, found `&String`

Reply:
Thank you, I must've compiled the wrong branch when I was initially testing this change.